base/agda/Free.agda

```diff
@@ -0,0 +1,12 @@
+module Free where
+
+-- free monad over shapes S and positions P
```

Comment:
The comments in the `.agda` files should also be proper sentences and formatted using Markdown.